source/common/common/BUILD

```diff
@@ -137,8 +137,14 @@ envoy_cc_library(
 # Contains minimal code for logging to stderr.
 envoy_cc_library(
     name = "minimal_logger_lib",
-    srcs = ["logger.cc"],
-    hdrs = ["logger.h"],
+    srcs = [
```

Comment:
split this to a separate target? this is no longer minimal with fancy logger

Reply:
In the next PR I plan to hook ENVOY_LOG with FANCY_LOG (with option provided to developers), and logically I think Fancy logger will be part of the logger as we need to include fancy_logger.h in that case?

Reply:
I plan to hook fancy logger with Envoy's logger so it's likely that I will include fancy_logger.h in logger.h. In that case I think it is also minimal_logger_lib?

Reply:
still, in the next PR you can just change their names?

Reply:
So you mean using another name like "fancy_logger_lib" for readability and keep current lib structure?

Reply:
@lizan Could you please give some clarification on which names you want me to change (or the lib structure)? Because the change of "minimal_logger_lib" seems unnecessary and very dirty. Thanks!

Reply:
Thanks this is now looks better.